### PR TITLE
i18n(de): update translation guides/customization

### PR DESCRIPTION
**Pull request by trueberryless**
#### Description

Translation of `guides/customization.mdx` into German language.

### CODE REVIEW

**Comment by trueberryless** [comment]
Link fixed in #2459

File: docs/src/content/docs/de/guides/customization.mdx [hunk 1]
```diff
@@ -77,21 +82,23 @@ Du kannst verschiedene Versionen deines Logos im Light- und Dark-Mode anzeigen.
...
    ```
 
+</Steps>
+
 ## Sitemap aktivieren
 
 Starlight hat eine eingebaute Unterstützung für die Erstellung einer Sitemap. Aktiviere die Sitemap-Generierung, indem du deine URL als `site` in `astro.config.mjs` angibst:
 
-```js
+```js {4}
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/customization.mdx:101]: I think this section is missing the changes from https://github.com/withastro/starlight/pull/2083/files#diff-596fab03bd0326b4a66f8afc7a97400b00fc458bdf93e1a32bcd72be7b8790f2

File: docs/src/content/docs/de/guides/customization.mdx [hunk 2]
```diff
@@ -276,6 +284,24 @@ hero:
...
+### Deaktivieren der Standard-404-Seite
+
+Wenn dein Projekt ein komplett angepasstes 404-Layout benötigt, kannst du eine `src/pages/404.astro`-Route erstellen und die [`disable404Route`](/de/reference/configuration/#disable404route) Konfigurationsoption setzen, um die Standard-Route von Starlight zu deaktivieren:
+
+```js {9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+export default defineConfig({
+	Integrationen: [
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/customization.mdx:296]:
```suggestion

export default defineConfig({
	integrationen: [
```

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://contribute.docs.astro.build/reviewers/reviewing-translations/). Left a few tiny comments, but except for that, it looks good to me.

File: docs/src/content/docs/de/guides/customization.mdx [hunk 3]
```diff
@@ -276,6 +286,25 @@ hero:
...
+
+Wenn dein Projekt ein komplett angepasstes 404-Layout benötigt, kannst du eine `src/pages/404.astro`-Route erstellen und die [`disable404Route`](/de/reference/configuration/#disable404route) Konfigurationsoption setzen, um die Standard-Route von Starlight zu deaktivieren:
+
+```js {9}
+// astro.config.mjs
+import { defineConfig } from 'astro/config';
+import starlight from '@astrojs/starlight';
+
+export default defineConfig({
+	integrationen: [
```
Comment (HiDeoo) [inline docs/src/content/docs/de/guides/customization.mdx:299]:
```suggestion
	integrations: [
```

Missed this one during my first review.

**Review by HiDeoo — APPROVED** [review]
Thanks for the updates, everything looks in order now :star2:

**Pull request merged**